cves/kernel/CVE-2021-45485.yml

```diff
@@ -55,7 +55,73 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: When computers talk to each other, they speak in packets.  A packet includes
```

Comment:
This is a really nice in-depth explanation of the vulnerability. However, I think you might want to summarize this information a little bit more. You can mention that an information leak flaw was found in the Linux kernel’s IPv6 implementation and then explain what that is. Then, the vulnerability in question which essentially allows a remote user to reveal sensitive information using a hash table in the IP ID generation.